### ISSUE TITLE
[new rule] Prefer test.todo()

### ISSUE THREAD

**Issue by macklinu** [jest-community/eslint-plugin-jest#217]
Introduced in Jest 24 (see [this blog post](https://jestjs.io/blog/2019/01/25/jest-24-refreshing-polished-typescript-friendly#testtodo)), it seems like we could support a rule that would prefer the following:

```js
test.todo('i need to write this test')
```

instead of:

```js
test('i need to write this test')
// and/or
test('i need to write this test', () => {})
```

Haven't thought much about specifics, rule naming, etc. Just came here from the [Jest 24 blog post](https://jestjs.io/blog/2019/01/25/jest-24-refreshing-polished-typescript-friendly#testtodo) and wanted to jot this down.

**Comment by doniyor2109** [comment]
Hi. Can I work on this?

**Comment by SimenB** [comment]
Yeah, go for it!

It should also warn against `test.skip('asdasdad', () => {})`
